src/js/views-dynamic-handler.js

```diff
@@ -1,46 +1,131 @@
 // For search functionality on the header.
 
-/* global Vue, axios, search, createPagination */
+/* global Vue, axios, search, createPagination, processDisplayResults, filter, $ */
 
+const pageSize = 10;
 const params = new URLSearchParams(window.location.search);
-let searchQuery = params.get("s");
-searchQuery = searchQuery ? searchQuery.toLowerCase() : undefined;
+let searchTerm = params.get("s") ? params.get("s").trim() : undefined;
```

Comment:
If Tag Filter is used and returns greater than 1 page of results then when you navigate to a different page number using pagination the term "undefined" is queried instead of taking you to the next page of results. Example:

1. Go to Views page
2. Use Tag Filter by only checking _test post_ (this gives multiple pages of results)
3. Click Apply filters
4. Scroll down to pagination
5. Use pagination to navigate to another page of results
6. See error (Instead of navigating to the next page of results "undefined" is being queried)

Reply:
Good catch, @TeddTech. This has been addressed and ready for another look. Thanks.

Reply:
Hi @cindyli, the error still persist.

Reply:
I'm having a weird problem that my new commit to my branch doesn't get shown on the pull request. Looking into it.

Reply:
Thanks for @jobara and @gtirloni's information, there's [a problem with Github Action](https://www.githubstatus.com/) at the moment. Let's see if new changes will be reflected when the problem is resolved.